omod/src/main/webapp/resources/htmlforms/section-history.xml

```diff
@@ -192,25 +195,25 @@
         <section id="complaint" sectionTag="fieldset" headerTag="legend" headerStyle="title"
```

Comment:
Hmmm, I know this isn't part of your addition, but "complaint" is a duplicate ID. We should fix at some point.